x/merkledb/node_test.go

```diff
@@ -19,14 +19,13 @@ func Test_Node_Marshal(t *testing.T) {
 	fullKey := ToKey([]byte("key"))
 	childNode := newNode(fullKey)
 	root.addChild(childNode, 4)
-	childNode.setValue(maybe.Some([]byte("value")))
+	childNode.setValue(SHA256Hasher, maybe.Some([]byte("value")))
 	require.NotNil(t, childNode)
 
-	childNode.calculateID(&mockMetrics{})
```

Comment:
`calculateID` probably used to populate the ID in these calls... but it doesn't do that anymore... Are these tests still doing what we want them to do?

Reply:
> Are these tests still doing what we want them to do?

I think so. We're marshaling a node and making sure that when we unmarshal it, we get the same thing back.